mmaction/datasets/pipelines/loading.py

```diff
@@ -178,6 +178,61 @@ def __call__(self, results):
         return results
 
 
+@PIPELINES.register_module()
+class UntrimSampleFrames(object):
```

Comment:
Untrim is verb so this name causes ambiguity. How about something like `SampleUntrimmedFrames` or `SampleFramesFromUntrimmed` or `UntrimmedSampleFrames`

Reply:
A question, can we reuse the class SampleFrames and remove this class?

Reply:
UntrimmedSampleFrames will be a good name.

Reply:
I think we would better not to reuse SampleFrames since SampleFrames is about sampling **num_clips** clips  from the video. In our case, the **num_clips** is unknown for an untrimmed video. We need to sample unknown numbers of clips from an untrimmed video of unknown length with a given distance.